src/widgets/WalletModal/WalletCard.tsx

```diff
@@ -13,11 +13,15 @@ interface Props {
 
 const StyledButton = styled(Button)`
   display: flex;
-  flex-direction: column;
-  justify-content: space-around;
-  padding: 8px 0 0;
+  justify-content: flex-start;
   white-space: nowrap;
-  min-height: 91px;
+  width: 100%;
+
+  background: ${({ theme }) => theme.colors.gray200};
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    width: 172px;
+  }
 `;
 
 const StyledText = styled(Text)`
```

Comment:
i thick this styled text may delete, Text component have "bold" and "color" props